controllers/passwords.js

```diff
@@ -45,3 +45,20 @@ export const handlePostPasswords = async (req, res) => {
   });
   sendResponse(res, password);
 };
+
+export const updatePassword = async (req, res) => {
+  const passId = req.params.id;
+  const { data } = req.body;
+  const encryptedData = cryptr.encrypt(data);
+  await updatePasswordSchema.validate({ ...req.body, id: passId });
+
+  await prisma.password.update({
```

Comment:
can we add any kind of checks to verify if update is successfull?

Reply:
<img width="531" alt="image" src="https://github.com/CodeWUsama/passwordvault-backend/assets/62501879/ad598e36-d798-40ba-99e2-03d408f5d2b8">
If there will be any issue while updating, `  sendResponse(res);` will not be executed instead try catch wrapper will act & throw an error to FE.